rules/possible-errors.js

```diff
@@ -32,14 +31,15 @@ module.exports = {
     'declaration-block-no-duplicate-properties': true, // Disallow duplicate properties within declaration blocks.
     'declaration-block-no-redundant-longhand-properties': true, // Disallow longhand properties that can be combined into one shorthand property.
     'declaration-block-no-shorthand-property-overrides': true, // Disallow shorthand properties that override related longhand properties within declaration blocks.
+    'declaration-block-no-duplicate-custom-properties': true, // disallow duplicate custom properties within declaration blocks
 
     // Block
     'block-no-empty': true, // Disallow empty blocks.
 
     // Selector
     'selector-pseudo-class-no-unknown': true, // Disallow unknown pseudo-class selectors.
     'selector-pseudo-element-no-unknown': true, // Disallow unknown pseudo-element selectors.
-    'selector-type-no-unknown': true, // Disallow unknown type selectors.
+    'selector-type-no-unknown': [true, {ignoreTypes: ["/^-styled-mixin/"]}], // Disallow unknown type selectors.
```

Comment:
so I just added this `ignoreTypes` option to it; the reason being is we had a [disabled line](https://bitbucket.foxsports.com.au/projects/MAR/repos/infinity/browse/src/components/molecules/nm/06-trig-btn/index.js#19) in infinity where it would error out as below:
![image](https://user-images.githubusercontent.com/13462129/140014376-1b382593-f4ef-4fa7-97c5-5e7983581c0d.png)

when I searched about the error saying `Unexpected unknown type selector "-styled-mixin0"` in [here](https://github.com/styled-components/stylelint-processor-styled-components/issues/268#issuecomment-495395414) adding this option (`{ignoreTypes: ["/^-styled-mixin/"]}`) fixes that error.

As far as the other error I just had to get rid of the space in infinity.